### ISSUE TITLE
Expose Exploitability and Impact Subscore of CVSS 3.1

### ISSUE THREAD

**Issue by Moneysac90** [Syslifters/sysreptor#55]
Currently, the `findings` Object does not expose the CVSS3.1 subscores "Exploitability" and "Impact". The following snipped shows an exemplary `findings` Object, which only stores the "cvss base score" named as `score`. 

```
{ "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:L/A:L", "score": "8.6", "level":"high", "level_number": 4 }
```

However, these scores are useful, if you want to use these values in your report/design or if you want to calculate your own scoring system. Implementation should be easy as these subscores are already calculated.

**Comment by aronmolnar** [comment]
Yes, this should not be a big deal.
We'll have a look at this.

**Comment by theguly** [comment]
i would also suggest to be ready for the upcoming CVSS 4.0 this fall ( https://www.first.org/cvss/v4-0/ )
i believe some people can quickly adapt to the new version of the standard (that looks cool!) while others will keep 3.1 for some time

**Comment by aronmolnar** [comment]
> i would also suggest to be ready for the upcoming CVSS 4.0 this fall ( https://www.first.org/cvss/v4-0/ ) i believe some people can quickly adapt to the new version of the standard (that looks cool!) while others will keep 3.1 for some time

We've already discussed this.

However, the metrics and calculations might still change and this will require more complex changes on our side.  
This is the reason we will still wait with v4

**Comment by aronmolnar** [comment]
The next release will provide you the following structure:

```
{
    "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N",
    "score": "0.00",
    "level": "None",
    "level_number": 0,
    "version": "3.1",
    "final": {
        "score": 0.00,
        "exploitability": 0.00,
        "impact": 0.00
    },
    "base": {
        "score": score_base,
        "exploitability": 0.00,
        "impact": 0.00
    },
    "temporal": {
        "score": 0.00,
        "exploitability": 0.00,
        impact": 0.00
    },
    "environmental": {
        "score": 0.00,
        "exploitability": 0.00,
        "impact": 0.00
    },
}
```

Those metrics are available for CVSS v2, v3.0 and v3.1.  
They will *not* be available for CVSSv4. Make sure, that in your design you check for the correct version.

**Comment by MWedl** [comment]
Implemented in https://github.com/Syslifters/sysreptor/releases/tag/2023.114
